lib/api/workout_repo.dart

```diff
@@ -5,7 +5,8 @@ import 'graphql.dart';
 
 abstract class WorkoutRepository {
   Future<Program> retrieveProgram();
-  Future<Workout> retrieveWorkout();
+
+  Future<Workout> mockRetrieveWorkout();
```

Comment:
just curious but why the change in name? I think it would've been ok to leave it since we would have to undo it once we implement the functionality after we implement it